README.md

```diff
@@ -6,88 +6,66 @@
 [![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status/Linux%20GPU%20CI%20Pipeline?label=Linux+GPU)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=12)
 [![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status/MacOS%20CI%20Pipeline?label=MacOS+CPU)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=13)
 
-**ONNX Runtime** is a cross-platform model **inferencing and training accelerator** compatible with most popular ML/DNN frameworks, including PyTorch, Tensorflow/Keras, scikit-learn, CoreML, and more.
+**ONNX Runtime** is a cross-platform **inferencing and training accelerator** compatible with many popular ML/DNN frameworks, including PyTorch, TensorFlow/Keras, scikit-learn, and more.
```

Comment:
nit: Can we please show some love to ML .NET? :) 

"including PyTorch, TensorFlow/Keras, scikit-learn, ML .NET and more.."